db/migrate/20220524040659_create_password_resets.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePasswordResets < ActiveRecord::Migration[6.0]
```

Comment:
What exactly does this class/migration do?  Please document somewhere.

Reply:
Yes, I updated the PR to include this schema change.